core/src/test/java/io/aiven/klaw/controller/RequestControllerTest.java

```diff
@@ -13,11 +13,8 @@
 import io.aiven.klaw.model.RequestVerdict;
 import io.aiven.klaw.model.enums.ApiResultStatus;
 import io.aiven.klaw.model.enums.RequestEntityType;
-import io.aiven.klaw.service.AclControllerService;
-import io.aiven.klaw.service.KafkaConnectControllerService;
-import io.aiven.klaw.service.RequestService;
-import io.aiven.klaw.service.SchemaRegstryControllerService;
-import io.aiven.klaw.service.TopicControllerService;
+import io.aiven.klaw.service.*;
```

Comment:
we import with wild card if imports are more than 20. It's a configuration we have in Intellij editor.